catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/config/ConfigurationApplication.java

```diff
@@ -173,6 +180,12 @@ public void setHiddenAttributes(List<String> hiddenAttributes) {
         this.hiddenAttributes = hiddenAttributes;
     }
 
+    public void setAttributeDescriptions(List<String> attributeDescriptions) {
+        this.attributeDescriptions = attributeDescriptions.stream()
+                .map(str -> str.split("="))
```

Comment:
What happens if there is an `=` in the attributeDescription?

Reply:
this was as all copied over from the existing attribute aliases code, so I guess I should add a fix there too?